daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/Pretty.scala

```diff
@@ -181,7 +181,7 @@ private[lf] object Pretty {
             )
           case DisclosurePreprocessing.NonExistentTemplate(templateId) =>
             text(
-              s"Template $templateId does not exist"
+              s"Template $templateId does not exist for the disclosed contract"
             )
```

Comment:
1. LookupError pretty return nicer messages
2. 
3. error message about lookup error are more consistent
4. We do not need to extend the Disclosure Error in the future if we rely on some other info in the package